index.html

```diff
@@ -7,58 +7,268 @@
     <link rel="stylesheet" href="Styles.css" />
     <title>Portfolio Website:Anthony</title>
```

Comment:
- [ ] This issue is not fixed yet, please take time to address all the required changes the previous reviewer mentioned
<img width="407" alt="1" src="https://user-images.githubusercontent.com/77843050/227741645-d31a990f-5d60-412d-a4ce-5030045ee6d0.png">